content/patterns/amd-rag-chat-qna/amd-rag-chat-qna-getting-started.adoc

```diff
@@ -0,0 +1,313 @@
+* Once the repository is cloned, select the folder where you cloned the repository (in the sidebar) and open `scripts/model-setup/upload-model.ipynb` jupyter notebook
+* Run this notebook by running each cell one by one
+** When prompted for a HuggingFace token, provide the prerequisite User Access Token and click `Login`
++
+image::/images/qna-chat-amd/rhoai-upload-model.png[Red Hat OpenShift AI - Run Jupyter notebook - Hugging Face token prompt, 50%, 50%]
++
+** When the notebook successfully runs, llama model should have been uploaded to either MinIO or AWS S3 under Llama-3.1-8B-Instruct directory in models bucket
++
+[NOTE]
+====
+By default, this notebook will upload the model to MinIO. To choose AWS S3, modify the last cell in the notebook by changing the value of XFER_LOCATION to AWS as shown below:
+----
+    XFER_LOCATION = 'MINIO'  <= current value
+    XFER_LOCATION = 'AWS'    <= modified to upload to AWS S3
+----
+====
+
+=== Deploy Model via Red Hat OpenShift AI
+
+Once the initial notebook has run successfully and the model is uploaded, you can deploy the model by following these steps:
+
+In the chatqna-llm data science project, select Models tab and click on Deploy model button and fill in the following fields as shown below:
+
+* Model name: `llama-31b`
+* Serving runtime: `vLLM AMD GPU ServingRuntime for KServe`
+* Model framework: `vLLM`
+* Deployment mode: `Advanced`
+* Model server size: `Small`
+* Accelerator: `AMD`
+* Model route: Enable `Make deployed models available through an external route` checkbox
+* Source Model location: Select `Existing connection` option
++
+--
+** Name: `model-store` (this is the name we used when we created the connection in `Create Connection` step)
+** Path: `Llama-3.1-8B-Instruct`
+*** Location where the model was copied to in the previous step
+--
++
+Click on Deploy to deploy this model. Once the model is successfully deployed, copy the inference endpoint to use it in the ChatQnA application (it will take a few minutes to deploy the model).
++
+[WARNING]
+====
+Make sure the model name is set to `llama-31b` as this is the value used in the deployment of llm micro-service that invokes the inference endpoint.
+====
+
+=== Deploy ChatQnA application
+
+This section provides details on installing the ChatQnA application as well as verifying the deployment and configuration by querying the application.
+
+Install ChatQnA app::
+
+Once all the prerequisites are met, install the ChatQnA application
+* Clone the repository by running the following commands:
++
+    git clone https://github.com/validatedpatterns-sandbox/qna-chat-amd.git
+    cd qna-chat-amd
++
+* Configure secrets for Hugging Face and inference endpoint
++
+    cp values-secret.yaml.template ~/values-secret-qna-chat-amd.yaml
++
+* Modify the `value` field in `~/values-secret-qna-chat-amd.yaml` file
++
+----
```

Comment:
```suggestion
[source,yaml]
----
```

Reply:
for some reason it didn't recognize the fix for this one, but it's in there.